downstream/modules/platform/ref-controller-credential-bitbucket.adoc

```diff
@@ -0,0 +1,7 @@
+[id="ref-controller-credential-bitbucket"]
+
+= BitBucket data center HTTP access token
+
+You can create Bitbucket Data Center HTTP access tokens for users and teams. 
+
+For further information, see link:https://confluence.atlassian.com/bitbucketserver/http-access-tokens-939515499.html[HTTP access tokens].
```

Comment:
Lila Yasin is going to review the draft I created for Bitbucket. [https://docs.google.com/document/d/1i-WW8qKPNpyfKvfoXz-QPPtcuBBDIt2OMh-tMhu5J7Q/edit?usp=drive_link](url)